modules/meshrenderinggl/glsl/fancymeshrenderer.frag

```diff
@@ -187,27 +188,31 @@ vec4 performShading() {
         // custom alpha
         float angle = abs(dot(normal, toCameraDir));
 #ifdef ALPHA_UNIFORM
-        alpha *= alphaSettings.uniformScale;
+        alpha *= max(alphaSettings.uniformScale, alphaSettings.minAlpha);
 #endif
 #ifdef ALPHA_ANGLE_BASED
         alpha *= pow(acos(angle) * 2 / M_PI, alphaSettings.angleExp);
+        alpha += alphaSettings.minAlpha;
 #endif
 #ifdef ALPHA_NORMAL_VARIATION
         float nv_dzi = dFdxFinest(normal.z);
         float nv_dzj = dFdyFinest(normal.z);
         float nv_curvature = min(1, nv_dzi * nv_dzi + nv_dzj * nv_dzj + 0.0000001);
         alpha *= min(1, pow(nv_curvature, alphaSettings.normalExp * 0.5) * 10);
+        alpha += alphaSettings.minAlpha;
```

Comment:
will this not give a max of 1+minAlpha? there are some off constants in this code :)

Reply:
Very true.